dist/index.html

```diff
@@ -28,79 +28,77 @@
         <span class="menu-btn__burger"></span>
       </div>
       <div class="nav-container lato_regular">
-        <img class="logo" src="./img/logo/logo-black.png" alt="" />
+        <a class="menu-link" href="./index.html"><img class="logo" src="./img/logo/logo-black.png" alt=""/></a>
```

Comment:
- [ ] Kindly add descriptive text to the `alt` attribute as this makes your page more accessible in case of image load failure.

Reply:
Fixed